Dockerfile

```diff
@@ -0,0 +1,75 @@
+# Copyright 2023 VMware, Inc. All Rights Reserved.
```

Comment:
Suggestion: Purely from user experience perspective, is it possible to rename this Dockerfile to be something else from which people understand this Dockerfile cannot be directly used by `docker build` under T-F root. (Or we could add some inline comments?)

The reason why I am raising this is because I got confused at the beginning until I looked into the dockerfile content.

Reply:
`docker-build` make target in t-f Makefile calls the `docker-build` target command in each component's Makefile, but I can add a comment to provide more clarity

Reply:
added the comment